### DISCUSSION TITLE
Npm registry unavailable

### DISCUSSION THREAD

**Discussion by jsirjacq** [community/community#48893] [Packages]
### Select Topic Area

Bug

- 502 Bad Gateway - PUT on NPM publish to my github registry
- Request failed "503 Service Unavailable" when I try to install a package previously published.

Has anyone faced the same issues recently?

**Comment by ramonjaspers** [comment]
yep, same here, seems to be a github issue :/ https://www.githubstatus.com/

**Reply by ramonjaspers** [reply]
Now its working for me! Hope it works for you too, gl

**Reply by jsirjacq** [reply]
Yep, seems to be working again!
